docs/get_started.md

```diff
@@ -10,3 +12,29 @@ run --f ../content/io/cloudslang/base/print/print_text.sl --i text=Hi
 ```
 
 + The CLI will run the ready-made `print_text` operation that will print the value passed to the variable `text` to the screen.
+
+# Docker
+
++ docker pull cloudslang/cloudslang
++ docker run -it cloudslang/cloudslang
++ At the prompt enter: 
+  
+```bash
+run --f ../content/io/cloudslang/base/print/print_text.sl --i text=Hi
+```
+
++ The CLI will run the ready-made `print_text` operation that will print the value passed to the variable `text` to the screen.
+
```

Comment:
You can add this plz:
Or 
docker run --rm -ti cloudslang run --f ../../content/io/cloudslang/base/print/print_text.sl --i text=first_flow

to run the flow without prompt...